src/NodaTime.Test/InstantTest.cs

```diff
@@ -16,20 +16,39 @@ public partial class InstantTest
         private static readonly Instant threeMillion = Instant.FromUntrustedDuration(Duration.FromNanoseconds(3000000L));
         private static readonly Instant negativeFiftyMillion = Instant.FromUntrustedDuration(Duration.FromNanoseconds(-50000000L));
 
+        // When dealing with floating point binary data, if we're accurate to 50 milliseconds, that's fine...
+        // (0.000001 days = ~86ms, as a guide to the scale involved...)
+
         [Test]
-        public void FromJDN()
+        public void FromJulianDate_OriginalTestData()
```

Comment:
The original tests were from Chapter 7 of Jean Meeus' [Astronomical Algorithms 2nd Edition](http://www.willbell.com/math/mc1.htm). Example 7.a gives a date and calculates a JD and example 7.b gives a JD and calculates a date. Example 7.c uses the same dates and JDs but reverses them.

There shouldn't be a problem with changing one or both, but personally I would keep one test far enough in the past that the Julian Calendar differs from the Gregorian by a few days.